tests/test_torch_compile.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import gym
+import torch as th
+
+from stable_baselines3 import SAC
+
+
+def test_load_compiled():

```

Comment:
Hello,
thanks for the PR, this test should only be run if pytorch version is >= 2.0
which is not SB3 minimal requirement for now.

Reply:
Skipping test on `Windows` platform or if PyTorch version smaller than "2.0.0".

Reply:
sounds good, although which package depends on packaging? (current SB3 doesn't depend on it)